Common/Inc/CommonDataTypes.hpp

```diff
@@ -18,6 +21,137 @@ typedef struct {
     float longitudeSpeed; //m/s
     double track; // degrees
     double heading; //degrees
-} LosSFData;
+};
+
+} // Namespace LOS
+
+namespace PM {
+
+struct JetsonToZpMovementCommand {
+  float x;
```

Comment:
do you want these to be dist_x dist_y as well or nah?

Reply:
I'm going to keep it as is, so it doesnt break any changes in PM. If neha wants it to be dist_x she can add that change to her PM PR.